src/linkedList/LinkedList.java

```diff
@@ -95,4 +95,70 @@ else if(this.tail.setNodeNext(someNode))
          return 0;
       }
    }
+   
+   public String toString()
+   {
+      ListNode curNode = this.head;
+      StringBuilder output = new StringBuilder();
+      output.append("This list of size " + this.length + " contains  \n");
+      if(this.head == null)
+      {
+         output.append("nothing\n");
+         return output.toString();
+      }
+      while(curNode != null)
+      {
+         output.append(curNode.toString() + "\n");
+         curNode = curNode.next;
+      }
+      return output.toString();
+   }
+//   public int addInOrderLargeFirst(ListNode someNode)
```

Comment:
I was thinking we for LinkedList and Queue that we should make a base class that is not sorted and then write SortedLinkedList and SortedQueue classes that inherit the base classes, but provided sorted adds and improved searching (binary search). These two functions you're potentially adding only work if the caller always uses these instead of the default add. If we have a doubly linked SortedLinkedList and SortedQueue then we don't need add smallest and add largest, just add. If they want largest ordering walk from tail. if want smallest ordering walk from head. So i'm thinking we delete these two functions. Thoughts?